From 79f496065bb2081ad3449f8dd2ca65b39c9129d7 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Fri, 9 Apr 2010 11:18:35 +0300
Subject: Fix CPANPLUS test failures when HOME doesn't exist

Bug: http://rt.cpan.org/Public/Bug/Display.html?id=52988
Bug-Debian: http://bugs.debian.org/577011
Origin: upstream

The Debian autobuilders are configured with a non-existing $ENV{HOME},
triggering a bug in CPANPLUS that causes test failures.

Fix from CPANPLUS-0.9001.

Patch-Name: fixes/cpanplus-without-home.diff
---
 cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
index 27d2abc..8475c36 100644
--- a/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
+++ b/cpan/CPANPLUS/lib/CPANPLUS/Internals/Utils.pm
@@ -5,7 +5,7 @@ use strict;
 use CPANPLUS::Error;
 use CPANPLUS::Internals::Constants;
 
-use Cwd qw[chdir];
+use Cwd                         qw[chdir cwd];
 use File::Copy;
 use Params::Check               qw[check];
 use Module::Load::Conditional   qw[can_load];
-- 
1.7.2.5

